src/modules/app/createNavigateToIntegreatUrl.js

```diff
@@ -22,7 +22,7 @@ export default (dispatch: Dispatch<StoreActionType>, navigation: NavigationScree
         createNavigateToEvent(dispatch, navigation)({ cityCode, language: newLanguage, path: pathname })
       } else {
         // '/augsburg/de/events'
-        createNavigateToEvent(dispatch, navigation)({ cityCode, language: newLanguage })
+        createNavigateToEvent(dispatch, navigation)({ cityCode, language: newLanguage, path: null })
```

Comment:
Minor: `undefined` would be more clear imo. Maybe we should find a rule of thumb of null vs undefined.

Reply:
Hmm... Yeah, I don't really like `undefined` though... Not sure what's best here; maybe avoiding it by using another field to distinguish `Events` and `Event`...
I'll leave it like this for now.